File: src/Models/Transaction.ts
export interface Transaction {
    total: number;
    tax: number;
    fees: number;
    paymentProcessor: string;
    currency: string;
    item_id?: number;
    item_name?: string;
    item_type?: string;
    direction: "in" | "out";
    date: Date;
    refunded: number;
    refundReason?: string;
    refundStatus?: string;
}

File: kotlin/com/example/lydaandroid/model/Transaction.kt
package com.example.lydaandroid.model

import java.util.Date

data class Transaction(
    val total: Double,
    val tax: Double,
    val fees: Double,
    val paymentProcessor: String,
    val currency: String,
    val item_id: Int? = null,
    val item_name: String? = null,
    val item_type: String? = null,
    val direction: String, // "in" | "out"
    val date: String,
    val refunded: Double,
    val refundReason: String? = null,
    val refundStatus: String? = null
)
